docs/reference/search/search-your-data/semantic-search.asciidoc

```diff
@@ -7,108 +7,86 @@ Semantic search is a search method that helps you find data based on the intent
+==== The {infer} API workflow
 
-Before you start using the deployed model to generate embeddings based on your input text, you need to prepare your index mapping first.
-The mapping of the index depends on the type of model.
-
-include::{es-ref-dir}/tab-widgets/semantic-search/field-mappings-widget.asciidoc[]
+The <<semantic-search-inference, {infer} API workflow>> is more complex but offers greater control over the {infer} endpoint configuration.
+You need to create an {infer} endpoint, provide various model-related settings and parameters, define an index mapping, and set up an {infer} ingest pipeline with the appropriate settings.
+The <<semantic-search-inference, Semantic search with the {infer} API>> tutorial shows you the process end-to-end.
 
 [discrete]
-[[semantic-search-generate-embeddings]]
-=== Generate text embeddings
-
-Once you have created the mappings for the index, you can generate text embeddings from your input text.
-This can be done by using an
-<<ingest,ingest pipeline>> with an <<inference-processor,inference processor>>.
-The ingest pipeline processes the input data and indexes it into the destination index.
-At index time, the inference ingest processor uses the trained model to infer against the data ingested through the pipeline.
-After you created the ingest pipeline with the inference processor, you can ingest your data through it to generate the model output.
+==== Model deployment workflow
 
-include::{es-ref-dir}/tab-widgets/semantic-search/generate-embeddings-widget.asciidoc[]
+You can also deploy NLP in {es} manually, without using an {infer} endpoint.
+This is the most complex and labor intensive workflow for performing semantic search in the {stack}.
+You need to select an NLP model from the {ml-docs}/ml-nlp-model-ref.html#ml-nlp-model-ref-text-embedding[list of supported dense and sparse vector models], deploy it using the Eland client, create an index mapping, and set up a suitable ingest pipeline to start ingesting and querying data.
+the <<semantic-search-deployed-nlp-model, Semantic search with a model deployed in {es}>> tutorial shows you the process end-to-end.
 
-Now it is time to perform semantic search!
 
 [discrete]
-[[semantic-search-search]]
-=== Search the data
+[[using-query]]
+=== Using the right query
 
-Depending on the type of model you have deployed, you can query rank features with a <<query-dsl-sparse-vector-query, sparse vector>> query, or dense vectors with a kNN search.
+Crafting the right query is crucial for semantic search.
+Which query you use and which field you target in your queries depends on your chosen workflow.
+If you're using the `semantic_text` workflow it's quite simple.
+If not, it depends on which type of embeddings you're working with.
 
-include::{es-ref-dir}/tab-widgets/semantic-search/search-widget.asciidoc[]
-
-[discrete]
-[[semantic-search-hybrid-search]]
-=== Beyond semantic search with hybrid search
+[cols="3*", options="header"]
```

Comment:
nit: The table columns are a bit disorienting, wonder if could give a bit more width to the first two?